src/components/Items/promItems.js

```diff
@@ -0,0 +1,82 @@
+import { items } from './data.js';
+import Image from 'next/image'; // Импорт компонент Image из next/image
+import './style.scss';
+import CartLogo from "@/public/icons/minicart.svg"
+import { Button } from '../shared/Button';
+
+export default function PromItems() {
+  return (
+    <div>
+      <strong><h2 style={{ textAlign: 'left', fontSize: '30px' }}><span style={{ color: '#FFBF00' }}>Акционные</span> товары</h2></strong>
+      <div className="item-container">
+        {items.map(item => (
+          <article className="item" key={item.id}>
+            <Image
+              src={item.imageId} // Используем путь из items
+              alt={item.name} // Используем название товара для alt
+              layout="responsive" // Устанавливаем адаптивный размер
+              width={100} // Ширина изображения
+              height={100} // Высота изображения
+            />
+            <div className="label">Популярное</div>
+            <div className = 'item-ml'>{item.ml} мл</div> {/* Добавляем надпись 450 мл */}
+            <div className="item-name">
+              <strong>{item.name.split(' ')[0]}</strong> {/* Первое слово жирным */}
+              {item.name.split(' ').slice(1).join(' ')} {/* Остальные слова */}
+            </div>
+
+            <div className="spacer"></div>
+            <div className="item-details">
+              Штрихкод: <span className="item-barcode">{item.barcode}</span> {/* Штрихкод */}
+            </div>
```

Comment:
```suggestion
            <p className="item-details">
              <strong className="item-barcode">Штрихкод:</strong>
              {item.barcode}
            </p>
```
лучше так реализовать подобный контент.